posts/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from .models import Post
+
+
+class PostSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Post
+        fields = ["photo", "description", "timestamp", "user"]
```

Comment:
For the `user` field: [Nested relationships in DRF](https://www.django-rest-framework.org/api-guide/relations/#nested-relationships)

Reply:
Done